app.js

```diff
@@ -774,7 +774,7 @@ function convertProfileToName(profile) {
 app.event("message", async ({ event, context, client, say }) => {
   try {
     // Filters for direct(instant) messages
-    if (event.channel_type === "im") {
+    if (event.channel_type === "im" && event.subtype !== "message_changed") {
```

Comment:
Every help request that was being submitted was erroring on this as it doesn't have an `event.text` property on it and was triggering this.

I've checked that saying `help` still works